### DISCUSSION TITLE
Add new items.

### DISCUSSION THREAD

**Discussion by moqsien** [avelino/awesome-go#5293] [Q&A]
## DevOps Tools

1. **vmr**([version-manger](https://github.com/gvcgo/version-manager)), a general version manager for multiple sdks.
2. **asciinema**([acast](https://github.com/gvcgo/asciinema)), a cross-platform asciinema(v2) terminal session recorder for MacOS/Linux/Windows.
3. **gobuilder**([gber](https://github.com/gvcgo/gobuilder)), an enhancement for official go build.

**Comment by phanirithvij** [comment]
#5295
#5294 
Please don't open issues and discussions. PR is enough.
